### DISCUSSION TITLE
Errors result to json

### DISCUSSION THREAD

**Discussion by sbitab** [directus/directus#2703] [Archive]
Hi, How to get json errors? I will send you an authentication request and if the login information is incorrect, the error **500 will return**.

I want something like this to return:
response status code: **200**
response body: 

```
{
  "error": {
     ....
  }
}
```

Can you help me?

**Comment by rijkvanzanten** [comment]
That's actually what Directus does. A wrong email/password request to `/auth/authenticate` results in

```json
{
  "error": {
    "code": 100,
    "message": "Invalid user credentials"
  }
}
```

You can try this yourself by sending a POST request to the demo API:

```
POST /thumper/auth/authenticate HTTP/1.1
Content-Type: application/json
Host: demo.directus.io
Connection: close
User-Agent: Paw/3.1.10 (Macintosh; OS X/10.15.4) GCDHTTPRequest
Content-Length: 66

{
  "email": "admin@example.com",
  "password": "wrong password"
}
```

The fact that it returns a 500 on your end indicates to me that there's a problem in the server or extensions. Are there any errors in the logs folder @sbitab?

**Comment by max-ra-gh** [comment]
Encountered the same issue, turns out it was caused by the `logger` setting having incorrect directory specified in the `config/project.php` file.
